src/app/objective/objective.component.ts

```diff
@@ -147,4 +152,23 @@ export class ObjectiveComponent implements OnInit {
         (this.objective!.resourceEstimate > 0 || this.objective!.assignments.length > 0) &&
         this.hasPeopleAvailable();
   }
+  displaySum() {
+    console.log("The sum is ");
+    let sum = Number(this.sum!) + Number(this.objective!.resourceEstimate)
+    console.log(sum);
+    this.cumiliativeSum.emit(sum);
+    return  sum;
+  }
+
+  setBackground() {
```

Comment:
This isn't a set method. It should be called something like `getCumulativeSumClass()`.

It should also have a return type annotation.